src/views/ImportFlow.vue

```diff
@@ -218,9 +218,23 @@ class ImportFlow extends mixins(Lang, Routes) {
       persistRoute: this.route('flows.persistFlowImport', {}),
       flowContainer: this.flowContainer,
     })
+    const numberOfFlows = get(flowContainer, 'flows.length', 0);
 
     if (flowContainer != null) {
       this.reset()
+      if (numberOfFlows > 1) {
+          this.$toast.success(`${Lang.trans('flow-builder.multiple-flows-saved', {
+          number: numberOfFlows,
+          })}`, {
+            timeout: 3000,
+            hideProgressBar: true,
+          })
+      } else {
+        this.$toast.success(`${Lang.trans('flow-builder.flow-saved')}`, {
+            timeout: 3000,
+            hideProgressBar: true,
+          })
+      }
```

Comment:
I'd suggest we format the message in the condition, and fire the toast after the condition
```
if () {
   message = trans('myMessage1')
} else {
   message = trans('myMessage2')
}

this.$toast.success(message, ...)
```

- [ ] refactor to call the toast once